### PR TITLE
Payment options changes for RIA-5719

### PR DESCRIPTION
**Pull request by sharmad-v1**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/postsubmit/AppealSavedConfirmation.java [hunk 1]
```diff
@@ -69,11 +68,7 @@ public PostSubmitCallbackResponse handle(
         AppealType appealType = asylumCase.read(APPEAL_TYPE, AppealType.class)
             .orElseThrow(() -> new IllegalStateException("AppealType is not present"));
 
-        if (appealType == AppealType.EA || appealType == AppealType.HU) {
-            paymentOption = asylumCase
-                .read(EA_HU_APPEAL_TYPE_PAYMENT_OPTION, String.class)
```
Comment (sajjadh-v1) [inline src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/postsubmit/AppealSavedConfirmation.java:74]: Do we not need to remove (del) the redundant EA_HU_APPEAL_TYPE_PAYMENT_OPTION?

File: src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/postsubmit/AppealSavedConfirmation.java [hunk 2]
```diff
@@ -82,7 +77,7 @@ public PostSubmitCallbackResponse handle(
         String submitPaymentAppealUrl = "";
         String payOrSubmitLabel = "";
 
-        if (paymentOption != null && paymentOption.equals("payNow")) {
+        if ((paymentOption != null && (paymentOption.equals("payNow")) || appealType == AppealType.EA || appealType == AppealType.HU)) {
```
Comment (sajjadh-v1) [inline src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/postsubmit/AppealSavedConfirmation.java:80]: Not sur eon the braces, looks like we have an additional pair, should simply be:
(paymentOption != null && (paymentOption.equals("payNow") || appealType == AppealType.EA || appealType == AppealType.HU))

**Review by sajjadh-v1 — DISMISSED** [review]

**Review by sajjadh-v1 — DISMISSED** [review]

**Review by rajeshthuraiyur — APPROVED** [review]
LGTM

**Review by sajjadh-v1 — APPROVED** [review]

**Review by rajeshthuraiyur — APPROVED** [review]
LGTM

**Pull request merged**